dataGen.js

```diff
@@ -100,1105 +12,43 @@ const genTime = () => {
 
 const commentDataGen = (string, songLength) => {
   const comments = [];
-
-  string = string.replace(/\.|,/g, '');
-
   let stringArr = string.split(' ');
-
-  for(let i = 1; i < 15; i++) {
-    const comment_id = 1;
+  
+  // construct comments array
+  for (let i = 1; i < 15; i++) {
+    const commentId = i;
     const max = 10;
-    const comment_text = [];
-    const user_image = '/.mary.png';
-    const comment_time = Math.floor(Math.random() * songLength);
+    const commentText = [];
+    const userImage = '/.mary.png';
+    const commentTime = Math.floor(Math.random() * songLength);
     for (let j = 0; j < max; j++) {
-      comment_text.push(stringArr.pop());
+      commentText.push(stringArr.pop());
     }
+    //assemble comments in an array
     comments.push({
-      comment_id: i,
-      comment_text: comment_text.join(' '),
-      user_image,
-      comment_time,
+      commentId,
+      comment_text: commentText.join(' '),
+      userImage,
+      commentTime,
     })
   }
   return comments;
```

Comment:
Do you intend to have userId or username as a key for the comments as well since you have userImage?

Reply:
The comment id is the primary key, but always in reference to the song in which the comment is nested.

Reply:
Cool!